src/components/v2/V2Create/tabs/ReviewDeployTab/FundingAttributes.tsx

```diff
@@ -0,0 +1,270 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { t, Trans } from '@lingui/macro'
+import { Statistic } from 'antd'
+import CurrencySymbol from 'components/shared/CurrencySymbol'
+import TooltipLabel from 'components/shared/TooltipLabel'
+import { formatWad } from 'utils/formatNumber'
+import {
+  formatDiscountRate,
+  formatRedemptionRate,
+  MAX_DISTRIBUTION_LIMIT,
+} from 'utils/v2/math'
+
+import FundingCycleDetailWarning from 'components/shared/Project/FundingCycleDetailWarning'
+import { detailedTimeString } from 'utils/formatTime'
+import {
+  DISCOUNT_RATE_EXPLANATION,
+  REDEMPTION_RATE_EXPLANATION,
+} from 'components/v2/V2Project/V2FundingCycleSection/settingExplanations'
+
+import { getBallotStrategyByAddress } from 'constants/v2/ballotStrategies/getBallotStrategiesByAddress'
+import {
+  FUNDING_CYCLE_WARNING_TEXT,
+  RESERVED_RATE_WARNING_THRESHOLD_PERCENT,
+} from 'constants/fundingWarningText'
+import { CurrencyName } from 'constants/currency'
+
+export function DistributionLimitStatistic({
+  distributionLimit,
+  currencyName,
+  showDetail,
+}: {
+  distributionLimit: BigNumber | undefined
+  currencyName: CurrencyName
+  showDetail?: boolean
+}) {
+  const hasDistributionLimit =
+    distributionLimit && !distributionLimit.eq(MAX_DISTRIBUTION_LIMIT)
```

Comment:
wonder if we should use `gte`

Reply:
Yeah could be smart, in case `MAX_DISTRIBUTION_LIMIT` changes and we're not ready